File: submissions/sm_033_surya/week_03/day_3/session_2/average_divisible.js
var sum = 0
var j = 0
var average
function average(limit,number) {
    

for (i = 1 ; i<= limit ; i++ )
{
    if (i % number == 0)
    {
        sum = sum+i
        j++
    }
    
}
average = sum / j 
    console.log(average)
}
average(50,5)

File: submissions/sm_015_lalit/week_13/day_4/session_1/average_divisible.py
userlimit=int(input("type the limit here : "))
userNum=int(input("type number now : "))
tcount=0
tsum=0
for i in range(1,userlimit+1):
    if i%userNum==0:
        tsum=tsum+i
        tcount=tcount+1

print(tsum/tcount)